components/TEventText.vue

```diff
@@ -61,10 +61,10 @@
         <div class="flex items-center gap-1">
           <TIcon name="lobby" class="w-4 h-4" />
           <div v-if="item.claimed === 'Yes'">
-            Organised by
+            {{$t('events.event.organizer')}}
```

Comment:
Please add spaces around like this, also in other places.

```suggestion
            {{ $t('events.event.organizer') }}
```